### ISSUE TITLE
ALMNH ES PI - Getting started

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/data-migration#110]
ALMNH Invert Paleo data was sent to Dave for "running through migrators". Needs to be scrubbed and prepped for upload to Arctos.

**Comment by Jegelewicz** [comment]
> Hi Teresa and Mariel,
> 
> I hope you are doing well. I’m embarrassed by how long it’s taken me to do this, but I’ve finished going through the error reports I received and have attached a significantly cleaned up version of our IZ database. There will definitely be a few things I’ve missed or ‘fixed’ in a way that will still cause errors, but this should be a big improvement. Is there any chance you could run this through the mill again before Teresa’s visit next week?
> 
> My main concern is getting these data online asap. Otherwise, some of the questions I wanted to discuss with you are:
> 
> 1) How to use ARCTOS to generate labels that could be printed by a datamax printer (I found this but am a bit overwhelmed: https://handbook.arctosdb.org/how_to/How-to-Create-Labels.html)
> 
> 2) How to best handle situations when we have a voucher specimen (say in 70% ethanol) and then multiple tissue samples in other fixatives and stored in different places – it seems my predecessors did all sorts of different things there…
> 
> 3) How to best handle old accession numbers when a specimen might have 2+ different old accession numbers
> 
> 4) I will be doing ship-based field work in Antarctica soon and am considering directly accessioning specimens into ARCTOS. Would it be possible to set up an instance on a laptop to work with on the ship and enter data using the fields or would it be better to just fill out a flat file? Would it be possible to have two laptops connected to the same networked instance so two folks could be doing data entry at the same time?
> 
> 5) I didn’t understand some of the errors and/or didn’t know how to fix them so I just skipped them – I will have a few questions about the remaining errors if you are able to run the data through the migrators again before Teresa’s visit.
> 
> I will be joining the call Friday morning so maybe some of these can be addressed then.
> 
> Thank you!

Kevin Kocot

**Comment by Jegelewicz** [comment]
> Hi Kevin,
> I am traveling out of the country, and I know Teresa is planning to be there shortly, but I just wanted to say that we can certainly help with these issues. The shipboard cataloging sounds great - and yes, you can have two computers and have simultaneous data entry by different users. Very cool! 
> Also, no problem to have a voucher in one place in one type of preservation and other parts elsewhere. These just become different specimen parts within the same catalog record. Eventually you may want to use object tracking and barcoding to locate them, but you can also just indicate location through part attributes or part remarks fields. 
> Each specimen gets one official accession number, so any others could be added as Other IDs or as part attributes. Teresa can look at your data and advise. 
> Good luck, and I'm sure Teresa will be in touch shortly.
> Best, 

Mariel

**Comment by Jegelewicz** [comment]
> Kevin,
> 
> I will try to look at this before I get there on Monday, but I cannot make any promises!  Here are some brief responses:
> 
> >My main concern is getting these data online asap.
> 
> I'll help with that to the best of my ability, but everything I do is volunteer time, so it may not go as quickly as hoped. I will do my best though! Also, Mary Beth can tell you, "running data through the migrators" does not prepare it for Arctos. I spent months getting the Vert Paleo data into Arctos AFTER it had "been through the migrators". I cannot apologize enough for the fact that you all were routed that direction, but here we are. I will do my absolute best to get things into Arctos quickly, but I am also looking for QUALITY data that you and the research community can use!
> 
> >Otherwise, some of the questions I wanted to discuss with you are:
> >1) How to use ARCTOS to generate labels that could be printed by a datamax printer (I found this but am a bit overwhelmed: https://handbook.arctosdb.org/how_to/How-to-Create-Labels.html)
> 
> I don't know that we have anyone with experience going from Arctos to Datamax. I will post up an issue on GitHub to see if I can find someone who has already done this, otherwise it will be up to us to figure it out AND write documentation! I (probably with Dusty's support) can help design the report if that is all that is needed.
> 
> >2) How to best handle situations when we have a voucher specimen (say in 70% ethanol) and then multiple tissue samples in other fixatives and stored in different places – it seems my predecessors did all sorts of different things there…
> 
> Mariel answered this one. "No problem to have a voucher in one place in one type of preservation and other parts elsewhere. These just become different specimen parts within the same catalog record. Eventually you may want to use object tracking and barcoding to locate them, but you can also just indicate location through part attributes or part remarks fields." The second option is what I did for Mary Beth in the vertebrate paleo collection.
> 
> >3) How to best handle old accession numbers when a specimen might have 2+ different old accession numbers
> 
> Mariel also gave an answer here. " Each specimen gets one official accession number, so any others could be added as Other IDs or as part attributes." The trick is that you need to decide which accession number is THE accession number. Mary Beth had this situation as well and the "old" numbers were documented as "original identifiers".
> 
> >4) I will be doing ship-based field work in Antarctica soon and am considering directly accessioning specimens into ARCTOS. Would it be possible to set up an instance on a laptop to work with on the ship and enter data using the fields or would it be better to just fill out a flat file? Would it be possible to have two laptops connected to the same networked instance so two folks could be doing data entry at the same time?
> 
> You cannot "set up an instance" of Arctos, however, if you have internet access you can enter data into Arctos. Otherwise, you will need to use the bulkloader and enter into an Excel file. There is a field data entry tool (Arctos Air) that was developed a while back, but I suspect it won't be the best alternative as it has not been updated for recent changes to the data structure. If you do have Internet access, you can have as many people as you like entering data into Arctos and you will have the additional advantage of being able to select some data from the controlled vocabularies rather than needing to check/change things later when they are misspelled or otherwise don't match the code tables.
> 
> >5) I didn’t understand some of the errors and/or didn’t know how to fix them so I just skipped them – I will have a few questions about the remaining errors if you are able to run the data through the migrators again before Teresa’s visit.
> 
> No worries - I'll check them out and we can work through them together.
> 
> Now for a few suggestions:
> 
> You should set up an account and get active on GitHub. (See Getting Started with GitHub for Arctos). It will be much more efficient if we communicate about all of this there so that Dusty and other Arctos users can chime in and help when I am not available (or don't have a good answer). We have an active group of paleo people and I think you will find that they can help you a lot!
> 
> Join the Arctos Working Group.
> 
> Have Mary Beth get you added to Arctos as an operator with access to your collection. I started a GitHub issue for this.
> 
> The first step in migrating data into Arctos is setting up your Agents. Many of them will already be in Arctos (Agents are shared by ALL Arctos collections), but we will probably need to add some. This will be the first thing that I scrutinize in the data you sent and I'll try to quickly get you a list of any people names that need clarification. I hope to send you that over the weekend.
> 
> Finally, you mentioned a call on Friday - I don't have anything on my calendar, so if I am missing something, please let me know when and by what method the call is scheduled for.
> 
> Adios,
> 

Teresa J. Mayfield-Meyer

**Comment by Jegelewicz** [comment]
> Hi all,
> 
> Teresa and Mariel, thank you very much(!) for the helpful answers. You’ll see that this database is still somewhat messy (disclaimer: I inherited and did not make this mess) but the key data should be there for virtually every specimen.
> 
> Just a couple quick responses:
> 
> We will not have internet access in Antarctica but I am interested in the bulkloader and Arctos Air. I will look into these before your visit and try to figure out if it could work for us. I’m also fine with us entering data directly into a spreadsheet but want to make sure that we’ve made it as easy as possible to import the data into Arctos after the cruise. In the past we hand-wrote data into notebooks but after the 20,000th tube handwriting is bad The cruise isn’t until November 2020.
> 
> I joined the Arctos working group and I already have a github account (under this e-mail address) – I commented on your issue so Mary Beth can easily find me.
> 
> I will look into Agents.
> 
> Maybe I was confused about Friday – that might just be a meeting of the folks here at UA. Mary Beth?
> 
> I will take some time do some more homework before we meet next week. I’m looking forward to it!
> 
> Thank you again!!!

Kevin

**Comment by Jegelewicz** [comment]
> Kevin,
> 
> No worries - EVERYONE's data is messy. Arctos will help remedy that!
> 
> Thanks for joining up - I also sent an invite for you to join the Arctos GitHub users team.
> 
> I started a data migration GitHub project just for your collection.
> 
> Please look at all of the in process issues. You will need to request a new collection portal before we can move any data into Arctos.
> 
> The bulkloader is an Excel spreadsheet with column headers that match data entry fields in Arctos. That will probably be the best tool for entering data on the ship, but we can discuss further next week.
> 
> Adios,

Teresa J. Mayfield-Meyer

**Comment by Jegelewicz** [comment]
@kmkocot I have a question! Which column of "catalog numbers" should be your Arctos catalog numbers? **MOLLUSK_catalog_id** or **catalog_id**

**Comment by Jegelewicz** [comment]
@kmkocot also, what is the significance of the numbers with decimals in the catalog_id column?

**Comment by kmkocot** [comment]
catalog_id

Thanks!



On Fri, Sep 27, 2019 at 12:18 PM -0500, "Teresa Mayfield-Meyer" <notifications@github.com<mailto:notifications@github.com>> wrote:


@kmkocot<https://github.com/kmkocot> I have a question! Which column of "catalog numbers" should be your Arctos catalog numbers? MOLLUSK_catalog_id or catalog_id

**Comment by kmkocot** [comment]
Those are pieces of tissue from the same individual as that catalog ID number. I'm not sure if that's the right way to do that though...



On Fri, Sep 27, 2019 at 12:18 PM -0500, "Teresa Mayfield-Meyer" <notifications@github.com<mailto:notifications@github.com>> wrote:


@kmkocot<https://github.com/kmkocot> I have a question! Which column of "catalog numbers" should be your Arctos catalog numbers? MOLLUSK_catalog_id or catalog_id

**Comment by Jegelewicz** [comment]
Thanks! I'm actually working on organizing the data into an Arctos format so we can look it over next week.

**Comment by Jegelewicz** [comment]
Data migration will be documented in project issues.
